content/en/docs/user-guides/how-to/plugin-management.md

```diff
@@ -11,22 +11,74 @@ Plugins for `notation` provide functionality and integration with key stores and
+## Usage
 
-{{% alert title="Warning" color="warning" %}}
-The following example only works for version 0.6.0 of the *notation-azure-kv* plugin on macOS with Apple Silicon using the default location for the Notation installation. You will need to update the filenames, location, and commands for other plugins, versions, and platforms. For more details on the default location of that directory on each platform, see [Notation directory structure for system configuration]({{< ref "/docs/user-guides/how-to/directory-structure.md" >}}).
+### Install a plugin from file system:
+
+`notation plugin install --file <file_path>`
+
+### Install a plugin from URL: 
+
+`notation plugin install --sha256sum <digest> --url <HTTPS_URL>`
+
+{{% alert title="plugin" color="info" %}}
+The following examples show how to install each plugin on a Linux AMD64 machine. To install a plugin on other operating systems and architectures, please get the URL or plugin binary file from plugin vendors' website.
 {{% /alert %}}
 
+## Install Notation AWS Signer plugin (v1.0.298)
+
+### Install from file system
+
+```console
+wget https://d2hvyiie56hcat.cloudfront.net/linux/amd64/plugin/latest/notation-aws-signer-plugin.zip
+
+notation plugin install --file notation-aws-signer-plugin.zip
+Successfully installed plugin com.amazonaws.signer.notation.plugin, version 1.0.298
+```
+Upon successful execution, the plugin is copied to Notation's plugin directory.
+
+### Install from URL:
+*(Installation from URL is unsupported.)*
+
+## Install Notation Azure Key Vault Plugin (v1.0.2)
+
+### Install from URL:
+
+```console
+notation plugin install --url https://github.com/Azure/notation-azure-kv/releases/download/v1.0.2/notation-azure-kv_1.0.2_linux_amd64.tar.gz --sha256sum f2b2e131a435b6a9742c202237b9aceda81859e6d4bd6242c2568ba556cee20e
+
+Downloading plugin from https://github.com/Azure/notation-azure-kv/releases/download/v1.0.2/notation-azure-kv_1.0.2_linux_amd64.tar.gz
+Download completed
+Successfully installed plugin azure-kv, version 1.0.2
+```
+
+### Install from local file:
+
+```console
+notation plugin install --file notation-azure-kv_1.0.2_linux_amd64.tar.gz
+Successfully installed plugin azure-kv, version 1.0.2
+```
+
+## Install Notation Venafi Plugin (v0.3.0)
+
+### Install from URL:
+
 ```console
-curl -Lo notation-azure-kv_0.6.0_darwin_arm64.tar.gz "https://github.com/Azure/notation-azure-kv/releases/download/v0.6.0/notation-azure-kv_0.6.0_darwin_arm64.tar.gz"
-mkdir -p ~/Library/Application\ Support/notation/plugins/azure-kv
-tar xvzf notation-azure-kv_0.6.0_darwin_arm64.tar.gz -C ~/Library/Application\ Support/notation/plugins/azure-kv notation-azure-kv
+notation plugin install --url https://github.com/Venafi/notation-venafi-csp/releases/download/v0.3.0/notation-venafi-csp-linux-amd64.tar.gz 
+Successfully installed plugin venafi-csp, version 0.3.0-release
```

Comment:
My understanding is that when installing via URL that the sha256 digest (--sha256sum) is required.

`notation plugin install --url https://github.com/Venafi/notation-venafi-csp/releases/download/v0.3.0/notation-venafi-csp-linux-amd64.tar.gz --sha256sum 03771794643f18c286b6db3a25a4d0b8e7c401e685b1e95a19f03c9356344f5a`

Reply:
Thanks for pointing out the oversight. I appreciate your review.